recipes/soxspipe/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "soxspipe" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 44dbc3057d71728cb62e4687093adf99184a59d055bc2a0fa8d9eeceee10235f
+
+build:
+  number: 0
+  entry_points:
+    - soxspipe=soxspipe.cl_utils:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - astropy
+    - bottleneck
+    - ccdproc
+    - docopt
+    - fundamentals
+    - matplotlib
```

Unless you need a pyqt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```